ocean_lib/models/test/test_data_nft_factory.py

```diff
@@ -640,8 +647,8 @@ def test_start_multiple_order(
             provider_data,
         ),
         (
-            consumer_wallet.address,
-            mock_dai_contract_address,
+            web3.toChecksumAddress(consumer_wallet.address.lower()),
```

Comment:
if these are calls to the models functions, then the conversion should be handled in models, not here in tests.

Reply:
ok.